doc/lua_api.txt

```diff
@@ -5428,6 +5439,7 @@ Authentication
     * The player needs to be online for this to be successful.
 
 * `minetest.get_auth_handler()`: Return the currently active auth handler
+    * Must be called *after* load time, returns `nil` otherwise
```

Comment:
Wouldn't it otherwise return `core.builtin_auth_handler`?

(Still, it's probably good to tell to only call this after the auth handler is properly set up (as long as the modder doesn't know what they're doing). :+1:)

Reply:
See https://github.com/minetest/minetest/blob/b7359f5fa93f531986c86b1ced20bb4584593b0a/builtin/game/auth.lua#L163-L165 btw..

Something like this:
```suggestion
    * Must be called *after* load time, to ensure that any custom auth handler was
      already registered.
```